### PR TITLE
Add /projects directory, and astrodb.markdown page to /project

### PR DESCRIPTION
**Pull request by ryanmaas**
Created  projects/astrodb.markdown page, and copied old AstroDB content over. Also modified current_projects to link to astrodb.html

### CODE REVIEW

**Comment by ryanmaas** [comment]
Fixes #21

File: projects/astrodb.markdown [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+# AstroDB
+
+## <a name="Overview" id="Overview"></a>Overview
+
+Astronomy has long been a data-intensive science. Today, the need to analyze massive amounts of data streaming from telescopes and generated by  simulations is more pressing than ever.
+
+ AstroDB  is an inter-disciplinary group bringing together faculty from the department of [Astronomy](http://www.astro.washington.edu/) (in particular the [Survey Science Group](http://ssg.astro.washington.edu/home_ssg.shtml) and the [N-body Shop](http://www-hpcc.astro.washington.edu/)), department of [Computer Science &amp; Engineering](http://www.cs.washington.edu/) (in particular the [Database Group](http://db.cs.washington.edu)), and the UW [eScience Instititute](http://escience.washington.edu/) to investigate new methods, tools, and techniques for data-driven discovery in Astronomy.
+
+
+## <a name="People" id="People"></a>People
```
Comment (domoritz) [inline projects/astrodb.markdown]: Do you want to delete the link?

File: projects/astrodb.markdown [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+*   [Magdalena Balazinska](http://www.cs.washington.edu/homes/magda/) (CSE and eScience)
+*   [Andrew Connolly](http://www.astro.washington.edu/users/ajc/) (Astronomy and eScience)
+*   [Julianne Dalcanton](http://www.astro.washington.edu/users/jd/Julianne_Dalcanton/Home.html) (Astronomy)
+*   [Jeffrey Gardner](http://www.phys.washington.edu/~gardnerj/) (Astronomy and eScience)
+*   [Bill Howe](http://www.cs.washington.edu/homes/billhowe/) (CSE and eScience)
+*   K. Simon Krughoff (Astronomy)
+*   Keith Wiley (Astronomy)
+*   [Benjamin Williams](http://www.astro.washington.edu/users/ben/) (Astronomy)
+
+&nbsp;
```
Comment (domoritz) [inline projects/astrodb.markdown:41]: Delete this

File: projects/astrodb.markdown [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+    **[A Study of Skew in MapReduce Applications](http://nuage.cs.washington.edu/pubs/opencirrus2011.pdf)**&nbsp;_(Best Student Paper)_. YongChul Kwon, Magdalena Balazinska, Bill Howe, and Jerome Rolia. In the 5th Open Cirrus Summit, Moscow, Russia, June, 2011.
+
+    **[Time Travel in a Scientific Array Database](http://scidb.cs.washington.edu/paper/ICDE13_conf_full_422.pdf)**. Emad Soroush and Magdalena Balazinska. ICDE 2013
+
+    **[ArrayStore: A Storage Manager for Complex Parallel Array Processing](http://scidb.cs.washington.edu/paper/sigmod362-soroush.pdf)**. Emad Soroush, Magdalena Balazinska, and Daniel Wang. SIGMOD 2011
+
+    **[Hybrid Merge/Overlap Execution Technique for Parallel Array Processing](http://scidb.cs.washington.edu/paper/soroush-array-workshop.pdf)**. Emad Soroush and Magdalena Balazinska, ArrayDB Workshop to be held in conjunction with EDBT 2011.
+
+### &nbsp;
```
Comment (domoritz) [inline projects/astrodb.markdown]: del

**Pull request merged**